src/bridge.ts

```diff
@@ -634,6 +635,9 @@ export class Bridge {
             }
             return undefined;
         });
+        this.appservice.on("ephemeral", async (event) =>
+            this.onEphemeralEvent(event as unknown as EphemeralEvent)
```

Comment:
Is this really guaranteed to be an `EphemeralEvent` or is `as unknown as X` the new `any`?

Reply:
It's guaranteed. `matrix-appservice-node` doesn't have types and will send us a Record<string, unknown> but that doesn't cast to a `EphemeralEvent` easily. The homeserver will only send us these events however.